src/components/dist/ProgressButton.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
```

Comment:
hmm? No idea where this came from. Anybody change anything in how project builds? I've runned it the same way I always had.

Reply:
I haven't changed anything in`tsconfig`, so idk